bokeh/server/server.py

```diff
@@ -128,7 +134,7 @@ def __init__(self, applications, io_loop=None, **kwargs):
             tornado_kwargs['redirect_root'] = kwargs.get('redirect_root', True)
 
             self._tornado = BokehTornado(self._applications, self.prefix, **tornado_kwargs)
-            self._http = HTTPServer(self._tornado, xheaders=kwargs.get('use_xheaders', False))
+            self._http = HTTPServer(self._tornado, xheaders=kwargs.get('use_xheaders', False), **tornado_server_kwargs)
```

Comment:
I might also suggest the name `http_server_kwargs` tho I would not object to keeping it as is either